### PR TITLE
chore(workflows): add macOS, Windows, node v13 and linting

### PR DESCRIPTION
**Pull request by ryanclark**
**What kind of change does this PR introduce?**
GitHub workflow changes

- Add macOS
- Add Windows
- Add Node 13
- Run linting tests
- Rename steps to be clearer

next - https://github.com/webpack/webpack-cli/pull/1355/

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -13,32 +13,35 @@ on:
...
       with:
         node-version: ${{ matrix.node-version }}
-    - name: yarn install, bootstrap
+    - name: Install dependencies
       run: |
         yarn
         yarn bootstrap
-    - name: Run test for webpack version ${{ matrix.webpack-version }}
+    - name: Linting
+      run: yarn lint
```
Comment (evilebottnawi) [inline .github/workflows/nodejs.yml]: Let's move it to own step, we don't need run link on every platform and node version

**Comment by evilebottnawi** [comment]
@rishabh3112 

From slack:

```
evilebottnawi:stuck_out_tongue:  5:35 PM
@sokra Can we integrate webpack-bot with github actions https://github.com/webpack/webpack-cli/pull/1356/?

sokra  5:49 PM
Yes, that would be the plan, but I need to find the time for that.

evilebottnawi:stuck_out_tongue:  5:50 PM
@sokra It’s normal that for now we will ignore it and fully integrate github actions for webpack-cli?

sokra  5:51 PM
yeah integrate with github actions.
```

So let's merge, I don’t think that the bot helps us so much, we see everything that comes from without a bot

**Comment by fokusferit** [comment]
May I ask what are you trying to do exactly? I see a lot of commits trying to skip tests.

**Comment by evilebottnawi** [comment]
@fokusferit Investigate why `execa` do not kill processes on GitHub actions, but works fine locally

**Comment by evilebottnawi** [comment]
Merging, feel free to feedback, in the next PRs, I will setup coverage normally

**Pull request merged**